packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.ts

```diff
@@ -101,15 +134,32 @@ export class WizardStepHeaderComponent extends CdkStepHeader implements AfterVie
         return returnedState;
     }
 
+    private getIconsForStates(icons: IWizardIcons, customIcons: IWizardIcons): IWizardIcons {
+        const stepIcons = icons;
+        if (customIcons) {
+            Object.keys(customIcons).forEach(icon => {
+                stepIcons[icon] = customIcons[icon];
+            });
+        }
+
+        return stepIcons
+    }
+
     _getDefaultIconForState(state: StepState): string {
         let returnedState: StepState = state;
 
-        switch (state) {
-            case "number":
-            case "done": returnedState = "step-complete"; break;
-            case "edit": returnedState = "edit"; break;
-            case "error": returnedState = "warning"; break;
-            default:
+        if (state === "number") {
+            returnedState = this.stepIcon || this.icons.initial;
+            this.iconColor = "gray";
+        } else if (state === "done") {
+            returnedState = this.icons.visited;
+            this.iconColor = "primary-blue";
+        } else if (state === "edit") {
+            returnedState = this.icons.selected;
+            this.iconColor = "black";
+        } else if (state === "error") {
+            returnedState = this.icons.error;
+            this.iconColor = "red";
```

Comment:
I don't see where iconColor is being used. Besides that, though, we should avoid setting it here because it's an unexpected side-effect of calling _getDefaultIconForState.